controllers/volumereplicationgroup_controller.go

```diff
@@ -707,7 +720,7 @@ func (v *VRGInstance) reconcileVRsForDeletion() bool {
 			continue
 		}
 
-		log.Info("Successfully processed VolumeReplication for PersistentVolumeClaim")
+		log.Info("Successfully protected PVC as secondary")
```

Comment:
(nit) log message should not state "Secondary" Deletion may have happened as either primary or secondary. Do we really need to change the older message?